tests/server/middlewares/managementApi.tests.js

```diff
@@ -0,0 +1,33 @@
+import { version as packageVersion } from '../../../package.json';
+import expect from 'expect';
+
+const managementApiClient = require('auth0-extension-express-tools').middlewares.managementApiClient;
+
+describe('ManagementApi middleware', () => {
+  let request = { user: { access_token: 'user_access_token' } };
+
+  function wrapperFunction(i, callback) {
+    return function() {
+      return callback(i);
+    };
+  }
+
+  it('should set header properly', () => {
+    const response = {};
+    managementApiClient({
+      domain: 'test-domain',
+      clientId: 'test-client',
+      clientSecret: 'test-secret',
+      headers: {
+        'User-agent': `${process.env.A0EXT_PROVIDER}-deploy-ext/${packageVersion} (node.js/${process.version.replace('v', '')})`
+      }
+    })(request, response, wrapperFunction(request, function(i) {
+      const keys = Object.keys(i.auth0);
+      keys.forEach(key => {
+        if (i.auth0[key].resource) {
+          expect(i.auth0[key].resource.restClient.options.headers['User-agent']).toEqual(`${process.env.A0EXT_PROVIDER}-deploy-ext/${packageVersion} (node.js/${process.version.replace('v', '')})`);
```

Comment:
Hmm, this is a test that is testing itself. It's not actually executing any of the logic `server/routes/webhooks.js`. Instead, it's actually testing the header setting login _within_ `auth0-extension-express-tools`. 

I have an idea:

What if we exported the managementApiClient created in `server/routes/webhooks.js` and then imported in this test to replace the client you create above on line 17? That might be a good middle ground.

Reply:
Decide drop this test because is irrelevant